### PR TITLE
feat: added list component

### PR DESCRIPTION
**Pull request by tanishq20**
Added list components

- Unordered list
- Ordered list
- Spaced list
- Stacked list

Live preview link: https://deploy-preview-35--compact-ui.netlify.app/docs/components/list

preview image

### CODE REVIEW

File: docs/components/list.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Compact UI | List</title>
```
Comment (Sameony) [inline docs/components/list.html:8]: do the naming int he format List | compact UI as the user will need to see the component's name first in the tab

**Review by rahulcs754 — APPROVED** [review]
Code is clean and good 
well define variable name

**Review by Sameony — APPROVED** [review]
great code, just a minor suggestion for title naming
you can merge it after looking it up

File: docs/components/list.html (hunk 1 above)
Comment (tanishq20) [reply docs/components/list.html:8]: Okay, will implement it.

**Pull request merged**